tensorrt_llm/_torch/speculative/interface.py

```diff
@@ -178,6 +185,9 @@ def create_cuda_graph_metadata(self, max_batch_size: int):
         cuda_graph_metadata.__post_init__()
         return cuda_graph_metadata
 
+    def is_layer_capture(self, layer_id: int):
+        return False
+
```

Comment:
This could use a docstring. Something that explains its relationship to `maybe_capture_hidden_states`